core-plugins/docs/WindowsShareCopy-action.md

```diff
@@ -1,43 +1,48 @@
-# Windows share copy action
+# Windows Share Copy
 
 
 Description
 -----------
-Copies a file or files on a windows share to HDFS directory
+Copies a file or files on a Microsoft Windows share to an HDFS directory.
 
 
 Use Case
 --------
-This action can be used when a file or files need to be moved from windows share to HDFS directory.
+This action can be used when a file or files need to be moved from a Windows share to an HDFS directory.
 
 
 Properties
 ----------
-**netBiosDomainName:** Specifies the NetBios domain name. Default is null. For example: `MyDomain`
+**netBiosDomainName:** Specifies the NetBios domain name. Default is null. For example: `MyDomain`. (Macro-enabled)
 
-**netBiosHostname:** Specifies the NetBios hostname to import files from. For example: `10.150.10.10`
+**netBiosHostname:** Specifies the NetBios hostname to import files from. For example: `10.150.10.10`. (Macro-enabled)
 
-**netBiosUsername:** Specifies the NetBios username to user for importing files from share.
+**netBiosUsername:** Specifies the NetBios username to use when importing files from the Windows share. (Macro-enabled)
 
-**netBiosPassword:** Specifies the NetBios password for importing files from windows share.
+**netBiosPassword:** Specifies the NetBios password to use when importing files from the Windows share. (Macro-enabled)
 
-**netBiosSharename:** Specifies the NetBios share name. For example: `share`
+**netBiosSharename:** Specifies the NetBios share name. For example: `share`. (Macro-enabled)
 
-**numThreads:** Specifies the number of parallel tasks to execute the copy operation. Defaults to 1.
+**numThreads:** Specifies the number of parallel tasks to use when executing the copy operation; defaults to 1. (Macro-enabled)
 
-**sourceDirectory:** Specifies the NetBios directory.For example: `/source/`
+**sourceDirectory:** Specifies the NetBios directory or file. For example: `source` or `source/file.log`. (Macro-enabled)
 
-**destinationDirectory:** The valid, full HDFS destination path in the same cluster where the file or files are to be
-moved. If a directory is specified with a file sourcePath, the file will be put into that directory. If sourcePath is a
-directory, it is assumed that destPath is also a directory. HDFSAction will not catch this inconsistency.
+**destinationDirectory:** The valid full HDFS destination path in the same cluster where
```

Comment:
These are docs, and are easier to read and review on GitHub if set to less than 120. Docs are set to  90 characters.